models/data_warehouse_l.model.lkml

```diff
@@ -119,13 +69,18 @@ explore: user_events_telemetry {
   }
 
   join: license_server_fact {
+    from: license_server_fact
+    view_label: "License Server Fact"
     relationship: many_to_one
-    sql_on: CASE WHEN ${user_events_telemetry._dbt_source_relation2}
-    IN ('"ANALYTICS".EVENTS.CLOUD_PORTAL_PAGEVIEW_EVENTS', '"ANALYTICS".EVENTS.PORTAL_EVENTS')
-    THEN COALESCE(${user_events_telemetry.portal_customer_id},${user_events_telemetry.context_traits_portal_customer_id})
-    = ${license_server_fact.customer_id}
-    ELSE ${user_events_telemetry.user_id} = ${license_server_fact.server_id} END
-    ;;
+    sql_on: ${user_events_telemetry.user_id} = ${license_server_fact.server_id} ;;
+  }
+
+  join: license_server_fact2 {
```

Comment:
We need this duplicate join on this table because we need to join portal data with product data. We do not need duplicate fields, just the join condition in the query. Its similar to a self-join but on a different field, which is `customer_id` in this case